test/unit/link.ts

```diff
@@ -0,0 +1,128 @@
+import { Links } from '../../src';
+import { expect } from 'chai';
+
+describe('Links object', () => {
+
+  it('should instantiate', () => {
+
+    const links = new Links('http://base.example/');
+    expect(links).to.be.an.instanceof(Links);
+
+  });
+
+  it('should allow adding links', () => {
+
+    const links = new Links('http://base.example/');
```

Comment:
As far as I understand this, are links now auto-instantiated with the base URL, and if a URL is not provided, it defaults to the base one of the original resource?

Reply:
Yes!